modules/nwtc-library/src/NWTC_IO.f90

```diff
@@ -4350,6 +4444,40 @@ SUBROUTINE PremEOF ( Fil , Variable, TrapErrors, ErrMsg )
 
    RETURN
    END SUBROUTINE PremEOF
+!=======================================================================
+   SUBROUTINE InitFileInfo( StringArray, FileInfo, ErrStat, ErrMsg )
+
+      CHARACTER(*), DIMENSION(:), INTENT(IN   ) :: StringArray
+      TYPE(FileInfoType),         INTENT(  OUT) :: FileInfo
+      INTEGER(IntKi),             INTENT(  OUT) :: ErrStat
+      CHARACTER(*),               INTENT(  OUT) :: ErrMsg
+
+      CHARACTER(*), PARAMETER :: RoutineName = 'InitFileInfo'
+      INTEGER :: i, n_lines
+
+      ErrStat = ErrID_None
+      ErrMsg  = ""
+
+      IF (.NOT. ALLOCATED(StringArray)) THEN
+         CALL SetErrStat( ErrID_Fatal, 'Attempting to use StringArray before it is allocated.' , ErrStat, ErrMsg, RoutineName )
+         RETURN
+      END IF
+
+      n_lines = size(StringArray)
+      FileInfo%NumLines = n_lines
+      FileInfo%NumFiles = 1
+
+      ALLOCATE( FileInfo%Lines(n_lines) )
+      ALLOCATE( FileInfo%FileLine(n_lines) )
+      ALLOCATE( FileInfo%FileIndx(n_lines) )
+      ALLOCATE( FileInfo%FileList(n_lines) )
```

Comment:
change to `ALLOCATE( FileInfo%FileList(1) )` since we only have one file.

Reply:
Ok good catch